user_modules/IsReady_Template/isReady.lua

```diff
@@ -257,7 +257,7 @@ end
 	end
 
 	commonPreconditions:BackupFile("sdl_preloaded_pt.json")
-	f = assert(io.open(config.pathToSDL.. "/sdl_preloaded_pt.json", "r"))
+	f = assert(io.open(commonPreconditions:GetPathToSDL().. "/sdl_preloaded_pt.json", "r"))
```

Comment:
@istoimenova, Please remove "/" from "/sdl_preloaded_pt.json" since it will be added by GetPathToSDL()